src/main.rs

```diff
@@ -244,7 +255,13 @@ fn go() -> Result<i32> {
         None => run_test(&container_name, &test_args),
         Some(n) => Err(Interrupted(n))
     };
-    try!(destroy(&container_name));
+
+    if no_destroy {
```

Comment:
IMO it'd be better to call `stop` unconditionally and then `destroy` if `! no_destroy`.